cfme-performance/tests/workloads/test_provisioning.py

```diff
@@ -0,0 +1,116 @@
+"""Runs Provisioning Workload."""
+from utils.appliance import clean_appliance
+from utils.appliance import get_server_roles_workload_provisioning
+from utils.appliance import set_server_roles_workload_provisioning
+from utils.appliance import wait_for_miq_server_ready
+from utils.conf import cfme_performance
+from utils.grafana import get_scenario_dashboard_url
+from utils.log import logger
+from utils.providers import add_providers
+from utils.providers import delete_provisioned_vm
+from utils.providers import delete_provisioned_vms
+from utils.providers import provision_vm
+from utils.providers import get_template_guids
+from utils.providers import get_remaining_vms
+from utils.smem_memory_monitor import SmemMemoryMonitor
+from utils.ssh import SSHClient
+from utils.workloads import get_provisioning_scenarios
+from itertools import cycle
+import time
+import pytest
+
+
+@pytest.mark.parametrize('scenario', get_provisioning_scenarios())
+def test_provisioning(request, scenario):
+    """Runs through provisioning scenarios using the REST API to
+    continously provision a VM for a specified period of time.
+    Memory Monitor creates graphs and summary at the end of each scenario."""
+
+    from_ts = int(time.time() * 1000)
+    ssh_client = SSHClient()
+    logger.debug('Scenario: {}'.format(scenario['name']))
+
+    clean_appliance(ssh_client)
+
+    scenario_data = {'appliance_ip': cfme_performance['appliance']['ip_address'],
+        'appliance_name': cfme_performance['appliance']['appliance_name'],
+        'test_dir': 'workload-provisioning',
+        'test_name': 'Test Provisioning',
```

Comment:
I think 'Provisioning' will suffice as a name.  This is used with the top header in the workload report in results directory.